### PR TITLE
ACS111113

### PR DESCRIPTION
**Pull request by darkness741852**

### CODE REVIEW

**Comment by darkness741852** [comment]
The modified code uses Tensorflow, which might be the reason the notebook execution failed (Tensorflow is currently incompatible with Python versions 3.10 or above, which might be what the Pytest is running on). The code runs perfectly fine on Google Colab, which should be evident by the contents of the .ipynb file.

**Comment by darkness741852** [comment]
Model Performance Summary:,
            Test Accuracy: 0.8513,
            Test Loss: 0.4689,
            Final Training Accuracy: 0.8382,
            Final Validation Accuracy: 0.8583,
            Final Training Loss: 0.4780,
            Final Validation Loss: 0.4399,
            Training Epochs: 20,
            Model Parameters: 552874

**Comment by rich7420** [comment]
viewed
